pkgs/top-level/python-packages.nix

```
@@ -813,7 +813,7 @@ in {
 
   mail-parser = callPackage ../development/python-modules/mail-parser { };
 
-  mailman = disabledIf (!isPy3k) (callPackage ../servers/mail/mailman/core.nix { });
+  mailman = disabledIf (!isPy3k) (callPackage ../servers/mail/mailman/default.nix { });
```

generally disable logic goes into the package itself.

also, if the filename is default.nix, you can just specify a directory
```suggestion
  mailman = callPackage ../servers/mail/mailman{ };
```